workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -92,21 +80,23 @@ public void Awake()
 
         public void Start()
         {
-            foreach (var worker in Workers)
+            foreach (KeyValuePair<SpatialOSWorld, ConnectionConfig> worldConfigPair in WorldToConfig)
             {
-                LoadLevel(worker);
+                LoadLevel(Vector3.zero);
```

Comment:
Whats the reason for always loading the world at the origin?

Reply:
no reason, forgot to change that :)